configuration/concepts/adult_hiv_followup.csv

```diff
@@ -0,0 +1,72 @@
+801a912f-fd72-43d5-8f5a-853be0812407, ,67d98a52-7749-4797-aa5b-315a3e8ce915;f04eaa0f-9a8a-479e-93f0-e2a5625ab759;4e134f44-0f34-4aea-ab29-cf287d6ae8f6,"Si échec, précisez le type","If unsuccessful, specify type ","Si échec, précisez le type","If unsuccessful, specify type ",Misc,Coded,,
+89720f63-0a65-4a5a-a3d5-8bafe5780644,,,Remarques (par ex- Préciser les molécules ARV pour lesquels le patient est résistant),Remarks (eg Specify the ARV molecules for which the patient is resistant),Remarques (par ex- Préciser les molécules ARV pour lesquels le patient est résistant),Remarks (eg Specify the ARV molecules for which the patient is resistant),Misc,Text,,
+fa9d9a5c-82e1-4865-8376-b9bf4e403994,0e86e2e4-d6e6-45af-9a60-496639e1c5e3;60f0d4e7-809a-47cb-a172-3fd68ed279ae;801a912f-fd72-43d5-8f5a-853be0812407;89720f63-0a65-4a5a-a3d5-8bafe5780644,,V. Modification du traitetement,V. Modification to treatment,V. Modification du traitetement,V. Modification to treatment,Misc,N/A,,
+dc10758d-06a2-43f1-b916-a111191e454a,,,Tuberculose pulmonaire (Date),Pulmonary tuberculosis ( Date),Tuberculose pulmonaire (Date),Pulmonary tuberculosis ( Date),Misc,Date,,
+e11eaac0-8c68-406e-aa78-34c2faf436d2,,,Zona (Date),Shingles (Date),Zona (Date),Shingles (Date),Misc,Date,,
+2a1c5417-6947-495b-a504-699eb08f224a,,,Tuberculose extra pulmonaire (Date),Extra-pulmonary Tuberculosis (Date),Tuberculose extra pulmonaire (Date),Extra-pulmonary Tuberculosis (Date),Misc,Date,,
+462e1a64-6d23-4590-9393-c5fdcba78a42,,,Retinite a CMV (Date),CMV Retinitis (Date),Retinite a CMV (Date),CMV Retinitis (Date),Misc,Date,,
+ecaf7d8e-161d-4e28-9ce8-b3b7dd9772bb,,,Sarcome de Kaposi (Date),Kaposi's sarcoma (Date),Sarcome de Kaposi (Date),Kaposi's sarcoma (Date),Misc,Date,,
+c6658a5c-91c9-4dd9-b9a2-d002a139dc97,,,Herpés (Date),Herpes (Date),Herpés (Date),Herpes (Date),Misc,Date,,
+20f7c749-9827-4508-a820-c134ebb3e271,,,Candidose oesophagienne (Date),Esophageal candidiasis (Date),Candidose oesophagienne (Date),Esophageal candidiasis (Date),Misc,Date,,
+175ae517-88f7-4c77-a07d-5bc149fcab3c,,,Isosporose (Date),Isosporosis (Date) ,Isosporose (Date),Isosporosis (Date) ,Misc,Date,,
+673ce6c2-9d44-48b2-8888-ef656cf1ccfd,,,Toxoplasmose cérébrale (Date),Cerebral Toxoplasmosis,Toxoplasmose cérébrale (Date),Cerebral Toxoplasmosis,Misc,Date,,
+e7800399-42d3-4dc6-af11-daa5f5682b2a,,,Cryptococcose neuro-méningée (Date),Cryptococcal neuro-meningeal disease (Date) ,Cryptococcose neuro-méningée (Date),Cryptococcal neuro-meningeal disease (Date) ,Misc,Date,,
+7b82d88a-f566-4934-b2ad-4f7c7bd48108,,,Méningite bactériennes (Date),Bacterial meningitis (Date) ,Méningite bactériennes (Date),Bacterial meningitis (Date) ,Misc,Date,,
+cb279713-3373-44be-91bf-ead37122d098,,,Autres Infections Oportunistes diagnostiquées,Other Oportunistic infections diagnosis,Autres Infections Oportunistes diagnostiquées,Other Oportunistic infections diagnosis,Misc,Text,,

```

Comment:
This is because Lisa added the wrong uuid for concept "Other". What should be done is to use the concept name "Other" and the right concept will be picked up. Although I think even better would be to use the concept "If other specify". I will chat to Lisa about this.